libs/k8s-client/src/ApiClient/PodsApiClient.php

```diff
@@ -31,22 +32,32 @@ public function getStatus(string $name, array $queries = []): Pod
         return $this->apiClient->request($this->baseApi, 'readStatus', Pod::class, $name, $queries);
     }
 
-    public function readLog(string $name, array $queries = []): string
+    public function readLog(string $name, array $queries = []): StreamInterface
     {
-        // do not use $this->request() wrapper, method does not return a class
-        $result = $this->baseApi->readLog($this->apiClient->getK8sNamespace(), $name, $queries);
+        $client = StreamClient::getInstance();
+        $response = $client->request(
```

Comment:
Tohle jsem koukam uplne nedomyslel :disappointed: Kdyz se ten request vytreli takle na primo, chybi tam standardni hangling z https://github.com/keboola/platform-libraries/blob/fb33c5cbce6949a5d985f3fec495fc25be53cf3a/libs/k8s-client/src/KubernetesApiClient.php#L59 (retries, standardni exceptions). Coz jsem uz stejne blbe udelal ja pred tim :facepalm: 

Samozrejme to nejde jen tak tam narvat - `clusterRequest` ocekava `AbstractAPI`. Ale par moznosti je. Asi nejlip, kdyz bysme si na to jeste zavovali.